src/component/absensi/shift/Shift.vue

```diff
@@ -19,74 +17,93 @@
                 </tr>
             </thead>
             <tbody>
-                <tr>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-
+                <tr v-for="item in shiftItems"
+                    :key="item.id"
+                    :item="item">
```

Comment:
in the shift page, we need to get the nik for updating the TapIn / TapOut. I think I missplace and using the wrong syntax. I have fix it in the last commit